packages/console/src/App.tsx

```diff
@@ -35,7 +35,9 @@ const Main = () => {
   const location = useLocation();
   const navigate = useNavigate();
   const fetcher = useSwrFetcher();
-  const { data } = useSWR<Setting, RequestError>('/api/settings');
+
```

Comment:
nit: unexpected blank line?

Reply:
I did it on purpose, to make settings specific logic separated from global